hazelcast/src/test/java/com/hazelcast/test/starter/ConfigConstructor.java

```diff
@@ -129,44 +124,42 @@ private static Object cloneConfig(Object thisConfigObject, ClassLoader classload
                     Object thisSubConfigObject = method.invoke(thisConfigObject, null);
                     updateConfig(setter, otherConfigObject, thisSubConfigObject);
                 } else if (returnType.getName().startsWith("com.hazelcast.memory.MemorySize")) {
-                    //ignore
+                    // ignore
                 } else if (returnType.getName().startsWith("com.hazelcast")) {
                     Object thisSubConfigObject = method.invoke(thisConfigObject, null);
                     Object otherSubConfig = cloneConfig(thisSubConfigObject, classloader);
                     updateConfig(setter, otherConfigObject, otherSubConfig);
-                } else {
-                    //
                 }
             }
         }
         return otherConfigObject;
     }
 
-    private static Class<?> getOtherReturnType(ClassLoader classloader, Class returnType)
-            throws ClassNotFoundException {
+    private static Class<?> getOtherReturnType(ClassLoader classloader, Class returnType) throws ClassNotFoundException {
         String returnTypeName = returnType.getName();
         if (returnTypeName.startsWith("com.hazelcast")) {
             return classloader.loadClass(returnTypeName);
         }
         return returnType;
     }
 
-    private static Method getSetter(Class otherConfigClass, Class returnType, String setterName) {
+    private static Method getSetter(Class<?> otherConfigClass, Class returnType, String setterName) {
         try {
             return otherConfigClass.getMethod(setterName, returnType);
         } catch (NoSuchMethodException e) {
+            return null;
         }
-        return null;
     }
 
     private static void updateConfig(Method setterMethod, Object otherConfigObject, Object value) {
         try {
             setterMethod.invoke(otherConfigObject, value);
         } catch (IllegalAccessException e) {
+            debug("Could not update config via " + setterMethod.getName() + ": " + e.getMessage());
         } catch (InvocationTargetException e) {
+            debug("Could not update config via " + setterMethod.getName() + ": " + e.getMessage());
         } catch (IllegalArgumentException e) {
-            System.out.println(setterMethod);
-            System.out.println(e);
+            debug("Could not update config via " + setterMethod.getName() + ": " + e.getMessage());
```

Comment:
Same question here. Now all exceptions are logged, but only if the `DEBUG_ENABLED` is set.